Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:18-alpine
+
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+
+COPY package.json yarn.lock ./
+
+RUN yarn
```

Comment:
Running `yarn` works. But I suggest using `yarn install` instead. Much more clear for beginners